includes/class-newspack-popups-model.php

```diff
@@ -210,52 +147,75 @@ public static function set_popup_options( $id, $options ) {
 	}
 
 	/**
-	 * Retrieve all inline popups.
+	 * Retrieve all overlay popups.
 	 *
-	 * @return array Inline popup objects.
+	 * @param  boolean       $include_unpublished Whether to include unpublished posts.
+	 * @param  array|boolean $group_slugs array Array of group slugs, or false to ignore groups.
+	 * @return array Overlay popup objects.
 	 */
-	public static function retrieve_inline_popups() {
+	public static function retrieve_overlay_popups( $include_unpublished = false, $group_slugs = false ) {
 		$args = [
 			'post_type'    => Newspack_Popups::NEWSPACK_POPUPS_CPT,
-			'post_status'  => 'publish',
+			'post_status'  => $include_unpublished ? [ 'draft', 'pending', 'future', 'publish' ] : 'publish',
 			'meta_key'     => 'placement',
 			// phpcs:ignore WordPress.DB.SlowDBQuery.slow_db_query_meta_value
-			'meta_value'   => self::$inline_placements,
+			'meta_value'   => self::$overlay_placements,
 			'meta_compare' => 'IN',
 		];
 
+		// If previewing specific groups.
+		if ( ! empty( $group_slugs ) ) {
+			$args['tax_query'] = [ // phpcs:ignore WordPress.DB.SlowDBQuery.slow_db_query_tax_query
+				[
+					'taxonomy' => Newspack_Popups::NEWSPACK_POPUPS_TAXONOMY,
+					'field'    => 'term_id',
+					'terms'    => $group_slugs,
+				],
+			];
+		}
+
 		return self::retrieve_popups_with_query( new WP_Query( $args ) );
 	}
 
 	/**
-	 * Retrieve all popups from a given group.
+	 * Retrieve all inline and above-header popups.
```

Comment:
Same nitpick as in the other place:
```suggestion
	 * Retrieve all inline popups.
```

Reply:
Updated in 6a5f9ed.